modules/ROOT/pages/restarting-a-node.adoc

```diff
@@ -18,61 +18,127 @@ Before any maintenance or restart of a node can be performed, it must be removed
 
 . Check if the firewalld service is running and enabled:
```

Comment:
@eldios are these steps for `firewalld` necessary? We already cover that during pre-install.
If not, please remove.